arcane/src/arcane/materials/srcs.cmake

```diff
@@ -46,6 +46,8 @@ set(ARCANE_SOURCES
   MeshMaterialInfo.cc
   MeshMaterialInfo.h
   MeshMaterialSynchronizer.cc
+  MeshMaterialSynchronizerImpl.cc
+  MeshMaterialSynchronizerImplAcc.cc
```

Comment:
Please rename `MeshMaterialSynchronizerImpl` to `LegacyMeshMaterialSynchronizerImpl` and `MeshMaterialSynchronizerImplAcc` to `AcceleratorMeshMaterialSynchronizerImpl`.